_integration-test/ensure-customizations-not-present.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -ex
 
-source "$(dirname $0)/../install/_lib.sh"
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
```

Comment:
I don't think this should need to be in each integration test, we have a test runner script where this might fit better. Also perhaps `INSTALL_SCRIPTS_DIR` is better, `SCRIPT_DIR` is a bit ambiguous.

Reply:
This expression evaluates to the path of the file it is written in. The whole point of it is not having to rely on some convention of how the scripts are arranged or from where they are called. I only ever use it once per script to refer to another script which then provides other constant values such as `PROJECT_ROOT`.

How else could we make references to other scripts more uniform and clear? Perhaps by always assuming we run from project root and define all paths as relative from there? That should solve the issue with the Docker build context too.

Alternatively, could we rename this variable to make its purpose clearer? `DIR_THIS_FILE_IS_IN`? Or maybe inline it, since it is only used once per file anyway?

Reply:
I removed this from every script, now everything always runs from project root and uses relative paths.